### ISSUE TITLE
Installation of the blender-import-lwo

### ISSUE THREAD

**Issue by Jaxtone** [douglaskastle/blender-import-lwo#22]
I am not a script or code skilled user of the old wreck Lightwave but I have an amount of already modeled 3D-Objects that would be perfect to import into Blender that definitely is a much better 3D-software than Lightwave ever could match.

So here are my questions and you may excuse me for being the guy who are better in guitar playing than scripting.

1. Where and how are the "blender-import-lwo-1.4.6" or "io_scene_lwo" supposed to be installed? 

2. Are they meant to be installed or placed in different folders?

3. About the map "blender-import-lwo-1.4.6.tar" I guess it´s the same as "blender-import-lwo-1.4.6" so I don´t need this if I already unpacked the "blender-import-lwo-1.4.6"? Right?

**Comment by harrier1231** [comment]
Ok I figured it out. It's pretty straight forward. For everyone else coming here with our question: Place the io_scene_lwo folder into the add-ons folder of blender. Then in blender go to edit-preferences and under add-ons enable the lwo addon (search for lightwave). It will now appear under file-import.

Came here with same question. My father, RIP, had a bunch of LWO objects and I want to see them! So I googled how to load LWOs without the insane price of lightwave and came to this but no clue how to make the add-on work. Did you ever get any insight? I'm not tech illiterate, but I'm also not savvy enough to figure this out.

**Comment by Donald-Drennan** [comment]
Yes, I got it to work finally. You have to have everything together in the
same folder.
